spec/facter/resolvers/processors_spec.rb

```diff
@@ -6,18 +6,31 @@
   end
 
   context 'when on x86 architecture' do
+    let(:log_spy) { instance_spy(Facter::Log) }
     let(:processors) { 4 }
     let(:models) do
       ['Intel(R) Xeon(R) CPU E5-2697 v4 @ 2.30GHz', 'Intel(R) Xeon(R) CPU E5-2697 v4 @ 2.30GHz',
        'Intel(R) Xeon(R) CPU E5-2697 v4 @ 2.30GHz', 'Intel(R) Xeon(R) CPU E5-2697 v4 @ 2.30GHz']
     end
     let(:physical_processors) { 1 }
+    let(:cores_per_socket) { 1 }
+    let(:threads_per_core) { 2 }
+    let(:lscpu_output) do
+      ["| grep -e 'Thread(s) per core': 2",
+       "-e 'Cores(s) per socket': 1"].join("\n")
+    end
```

Comment:
I still fail to understand how this is something that `lscpu` outputs. Output should be similar to this:

```
2.6.3 :001 > require 'facter'
 => true 
2.6.3 :002 > Facter::Core::Execution.execute("lscpu | grep -e 'Thread(s)' -e 'Core(s)'")
 => "Thread(s) per core:              2\nCore(s) per socket:              4" 
```